Automattic/wp-calypso
# Gutenberg: update npm packages to latest versions

#### Changes proposed in this Pull Request

Updates Gutenberg npm packages to latest versions. 

It seems like this update will not require as much fixes as the ones before it. :)

#### Testing instructions

1. Navigate to http://calypso.localhost:3000/gutenberg/
2. Verify that there are no new errors in console.
3. Try inserting some content and verify that publishing still works.
4. Smoke test Calypso and check bundle sizes.

_On hold until new version of components package, that contains WordPress/gutenberg#10943 fix, is published._
